features/ultils/helper.ts

```diff
@@ -36,10 +35,17 @@ export const getCliKintoneBinary = (): string => {
   }
 };
 
-export const replaceTokenWithEnvVars = (input: string) =>
-  input.replace(/\$\$[a-zA-Z0-9_]+/g, replacer);
+const replaceTokenWithEnvVars = (
+  input: string,
+  envVars: { [key: string]: string } | undefined,
+) =>
+  input
+    .replace(/\$\$[a-zA-Z0-9_]+/g, processEnvReplacer)
+    .replace(/\$[a-zA-Z0-9_]+/g, (substring) =>
+      worldEnvReplacer(substring, envVars),
+    );
```

Comment:
NOTE: You can write this shorter using currying.

```suggestion
    .replace(/\$[a-zA-Z0-9_]+/g, worldEnvReplacer(envVars)(substring));
```

Reply:
@mshrtsr 

I have tried, but it does not work as expected.
Could you send the complete example?

Reply:
@hung-cybo 
Define `inputEnvReplacer` as follows.

```typescript
const inputEnvReplacer = (envVars: { [key: string]: string } | undefined) => {
  return (substring: string) => {
    if (!envVars) {
      return substring;
    }

    const key = substring.replace("$", "");
    const value = envVars[key];
    if (value === undefined) {
      throw new Error(`The env variable in input parameter is missing: ${key}`);
    }
    return value;
  };
};
```

We use this technique in the transformer of `RecordSchema`.
https://github.com/kintone/cli-kintone/blob/main/src/record/export/schema/transformers/formLayout.ts